pom.xml

```diff
@@ -101,6 +101,25 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>com.mycila</groupId>
+          <artifactId>license-maven-plugin</artifactId>
+          <version>3.0</version>
+          <configuration>
+            <mapping>
+              <gss>JAVADOC_STYLE</gss>
+              <java>SLASHSTAR_STYLE</java>
```

Comment:
I don't have a problem with the JAVADOC_STYLE.  It's just not the style that was used before